ansible/roles/aws/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# change it if you want a specific version
+docker_apt_repository: 'deb [arch=amd64] https://download.docker.com/linux/ubuntu/ {{ ansible_distribution_release }} stable'
```

Comment:
please remove this variable. It is not needed

Reply:
fixed in the last commit